v4/source/saml.yaml

```
@@ -252,3 +252,51 @@
           $ref: "#/components/responses/Forbidden"
         "501":
           $ref: "#/components/responses/NotImplemented"
+  /saml/reset_auth_data:
+    post:
+      tags:
+        - SAML
+      summary: Reset AuthData to Email
+      description: >
+        Reset the AuthData field of SAML users to their email. This is meant to be
+        used when the "id" attribute is set to an empty value ("") from a previously
+        non-empty value.
+
+        ##### Permissions
+
+        Must have `manage_system` permission.
```

Add minimum server version. 

```suggestion
        Must have `manage_system` permission.
        
        __Minimum server version__: 5.35
```